### PR TITLE
Update design of creation to be consistent with mockups

### PR DESCRIPTION
**Pull request by NicuPascu**
# Description

See [JENKINS-46111](https://issues.jenkins-ci.org/browse/JENKINS-46111), [JENKINS-45145](https://issues.jenkins-ci.org/browse/JENKINS-45145)

# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

File: blueocean-dashboard/src/main/js/creation/bitbucket/BbIcon.jsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import React from 'react';
...
...11019,22.8192626 16.0811771,23.9998301 12.3958761,24 L12.3957062,24 L12.3955363,24 C8.71040523,23.9998301 5.65031049,22.8192626 5.30346463,21.2549725 C5.25248288,21.0253847 4.96222677,19.5164947 4.56490898,17.3898759 C4.55947092,17.3609862 4.55658196,17.3314168 4.55658196,17.3020173 C4.55658196,17.0219576 4.73552791,16.8951829 4.92433033,16.8951829 C5.08220383,16.8951829 5.20897845,17.0070029 5.20897845,17.0070029 C5.20897845,17.0070029 7.76877226,19.0342073 12.3957062,19.0343773 C17.0224702,19.0342073 19.582434,17.0070029 19.582434,17.0070029 C19.582434,17.0070029 19.7090386,16.8951829 19.8670821,16.8951829 Z M12.3955193,14.8742663 C10.5705425,14.8742663 9.09105199,13.3946058 9.09105199,11.569799 C9.09105199,9.74465223 10.5705425,8.2653317 12.3955193,8.2653317 C14.2204961,8.2653317 15.6999865,9.74465223 15.6999865,11.569799 C15.6999865,13.3946058 14.2204961,14.8742663 12.3955193,14.8742663 Z M12.3923244,9.91273907 C11.4777118,9.91273907 10.7362671,10.6541837 10.7362671,11.5687963 C10.7362671,12.483409 11.4777118,13.2248536 12.3923244,13.2248536 C13.3069371,13.2248536 14.0483817,12.483409 14.0483817,11.5687963 C14.0483817,10.6541837 13.3069371,9.91273907 12.3923244,9.91273907 Z" />
```
Comment (vivek) [inline blueocean-dashboard/src/main/js/creation/bitbucket/BbIcon.jsx]: Probably its fine but noticed ticket saying:

>Bitbucket #0e2b4f
>Everything else #494a49

but fill is `#4A4A4A`?

Comment (NicuPascu) [inline blueocean-dashboard/src/main/js/creation/bitbucket/BbIcon.jsx]: The colours where indeed inconsistent, I changed them so now they match the design

**Comment by michaelneale** [comment]
once build is green then :bee:

**Comment by i386** [comment]
@NicuPascu best post these with a screenshot. Takes me a while to switch between branches. 

🐝

**Comment by i386** [comment]
:bee:

**Comment by NicuPascu** [comment]
@i386 will do in the future, thanks

**Comment by michaelneale** [comment]
:bee:

**Pull request merged**